src/pages/documentation/experience-cloud-extensions.md

```diff
@@ -5,6 +5,7 @@ Experience Cloud extensions are extensions that directly connect with Experience
 ## Extensions
 
 - [Adobe Analytics](./adobe-analytics/index.md)
+- [Adobe Streaming Media for Edge Network](./media-for-edge-network/index.md)
```

Comment:
Is this needed here? This document is for "extensions that directly connect with Experience Cloud solution", however Streaming Media for Edge Network does not as it connects to the Edge Network.

Reply:
Looking at the AdobeDocs, this makes sense to me now.